app/move/app/new/controllers/base.js

```diff
@@ -225,6 +228,36 @@ class CreateBaseController extends FormWizardController {
 
     return false
   }
+
+  // TODO: Temporary data collection to understand impact of
+  // adding PNC number validation
+  validateFields(req, res, callback) {
+    super.validateFields(req, res, errors => {
+      const PNCPredicate = error =>
+        error.type === 'policeNationalComputerNumber'
+
+      forEach(pickBy(errors, PNCPredicate), (error, key) => {
+        Sentry.captureException(new Error('PNC validation failed'), {
+          level: Sentry.Severity.Warning,
+          tags: {
+            'validation_error.key': key,
+            'validation_error.path': error.url,
+            'validation_error.type': error.type,
```

Comment:
Should we capture the failed input value (either here or in the context)?

I've left it off initially to get an idea of how often it occurs first, and we can also run a query on the DB to get an idea of failed patterns as a first, next step.

Reply:
Agreed, I think it's okay to leave it off for now, I'd assume we don't get the same ones appearing over and over anyway.